sopel/modules/announce.py

```diff
@@ -9,14 +9,24 @@
 """
 from __future__ import unicode_literals, absolute_import, print_function, division
 
+from builtins import range
+
 from sopel.module import commands, example, require_admin
 
 
+# from https://stackoverflow.com/a/312464/5991 with modified names for readability
```

Comment:
Put that in a proper docstring and I'm sold.

Reply:
Why would you want this in a docstring? I can understand wanting **a** proper docstring for the function, but nobody needs to see the SO source in a hypothetical documentation page (not that `modules` gets doc'd).

Reply:
Yeah, it's me, I always assume I shouldn't have to read the source code to get everything I want from a documentation. In that case, maybe we don't need a docstring.

But sometimes, I have to read the source code. In that case, I tend to scan for `def`, then I read under it, not above it. So, middle ground here: can you put that under the `def chuncks` line instead of above?